src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Authentication;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods for setting up cookie authentication services in an <see cref="IServiceCollection" />.
+    /// </summary>
+    public static class CookieAuthenticationServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Adds cookie authentication services to the specified <see cref="IServiceCollection" />. 
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
+        /// <returns>A reference to this instance after the operation has completed.</returns>
+        public static IServiceCollection AddCookieAuthentication(this IServiceCollection services)
+        {
+            if (services == null)
+            {
+                throw new ArgumentNullException(nameof(services));
+            }
+
+            services.AddAuthentication();
+            services.AddAntiforgery();
+            return services;
+        }
+
+        /// <summary>
+        /// Adds cookie authentication services to the specified <see cref="IServiceCollection" />. 
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
+        /// <param name="configureOptions">An action delegate to configure the provided <see cref="SharedAuthenticationOptions"/>.</param>
+        /// <returns>A reference to this instance after the operation has completed.</returns>
+        public static IServiceCollection AddCookieAuthentication(this IServiceCollection services, Action<SharedAuthenticationOptions> configureOptions)
```

Comment:
He just renamed this from AddAuthentication(SharedAuthenticationOptions) and added anti-forgery. Without it, you have to do:

services.AddAntiforgery();
services.AddAuthentication(sharedOptions => ...);

I'm with pinpoint honestly this feels super gross with this new name. I think once we refactor things into a base so this can continue to be called AddAuthentication things will be better.